color_refinement.py

```diff
@@ -183,6 +184,153 @@ def my_test(cg):
     fast_color_refine(cg)
 
 
+def choose_a_root(g: Graph) -> Vertex:
```

Comment:
Moet dit allemaal in color_refinement? Wat mij betreft maken we een file specifiek voor trees.

Reply:
Vind ik helemaal prima :)